design/tokens/theme.ts

```diff
@@ -4,6 +4,8 @@ import breakpoints from './breakpoints';
 import components from './components';
 import styles from './styles';
 
+// This removes unwanted components from the default theme
+const filteredDefaultComponents = Object.fromEntries(Object.entries(defaultTheme.components).filter(([key]) => !['Menu'].includes(key)))
```

Comment:
Not my favorite thing in the world, but because of the defaults we need to remove the default theme for Menu. This will need to be removed in the future with styling for the Menu component is moved to this library (Theming was added recently)